Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ContestsController.cs

```diff
@@ -65,11 +61,7 @@ public ActionResult Index()
         [HttpGet]
         public ActionResult Create()
         {
-            var newContest = new ViewModelType
-            {
-                SubmissionTypes = this.Data.SubmissionTypes.All().Select(SubmissionTypeViewModel.ViewModel).ToList()
-            };
-
+            var newContest = new ViewModelType();
```

Comment:
If you won't add any data to the view model, you don't need to initialize a new variable for it. You just can return `this.View(new ViewModelType());`